CHANGES.rst

```diff
@@ -4,7 +4,9 @@ astropy-helpers Changelog
 4.0 (unreleased)
 ------------------
 
-- No changes yet.
+- Changed minimum required Python version to 3.6. [#498]
```

Comment:
would also mention the change in the minimum sphinx version

Reply:
Done